### DISCUSSION TITLE
What is the WeibullATF rho and how is it estimated?

### DISCUSSION THREAD

**Discussion by gabgilling** [CamDavidsonPilon/lifelines#1382] [Q&A] [answered]
Hello,

I've been going through the lifelines documentation as I'm trying to learn about survival models and their applications.

When discussing the Weibull Hazard function, the documentation writes down its Hazard function and includes a parameter rho, but does not explain how it is being estimated. Would you mind explaining how it is estimated?

Thank you

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi there,
Estimation of rho is done like any other parameter: minimizing the neg-log-likelihood (it's an unknown, just like the coefficients in the additive part). Rho can _also_ be expressed as an additive equation, like in the docs [here](https://lifelines.readthedocs.io/en/latest/fitters/regression/WeibullAFTFitter.html), and again, the estimation is done via minimization. 

Does that answer your question?

**Reply by gabgilling** [reply]
Thank you for your prompt response! I had missed that page in the documentation.
Would you mind specifying what likelihood function you're using for rho? Or are you simply taking the derivative of the survival function with regards to rho and solving for that?

Thanks again

**Reply by CamDavidsonPilon** [reply]
The likelihood is complicated by the censoring of the data, but it does come down to minimizing the neg-log-likelihood via some sort of gradient descent. 

The likelihood can be seen [here](https://cran.r-project.org/web/packages/flexsurv/vignettes/flexsurv.pdf), section 2.

**Reply by gabgilling** [reply]
That's just what I was looking for, thank you!
